File: MpAndroidChart/app/src/main/java/com/example/jestina/mpandroidchart/view/MainActivity.java
package com.example.jestina.mpandroidchart.view;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.jestina.mpandroidchart.R;
import com.example.jestina.mpandroidchart.contract.MainContract;
import com.example.jestina.mpandroidchart.presenter.MainPresenter;

public class MainActivity extends AppCompatActivity implements MainContract.View {
    MainPresenter mainPresenter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        initView();
        initPresenter();
    }

    @Override
    public void initView() {
        setContentView(R.layout.activity_main);
    }

    @Override
    public void initPresenter() {
        mainPresenter = new MainPresenter();
    }
}

File: CountUpAni_Kt/app/src/main/java/com/example/jestina/countupani_kt/view/MainActivity.kt
package com.example.jestina.countupani_kt.view

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.example.jestina.countupani_kt.R
import com.example.jestina.countupani_kt.utils.YWLog

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        YWLog.d("adsf")
    }
}
